### ISSUE TITLE
Build it by myself, but not work.

### ISSUE THREAD

**Issue by liweilong** [fkorotkov/intellij-thrift#128]
I had build the latest code for myself and install from the disk, but it not work.
Do you have commit to commit update to the plugin repo? It have been for many days no see any review.

**Comment by Gibbonator** [comment]
You can just remove the repository entry for bintray and it will build. I think the provided dependencies are just for the CI github integration

**Comment by liweilong** [comment]
> You can just remove the repository entry for bintray and it will build. I think the provided dependencies are just for the CI github integration

Yes, I had did that and install from local, but there are no highlight for thrift file.

**Comment by clatisus** [comment]
@liweilong Are you installing from the `thrift.zip` under `thrift/build/distributions`? I've tried that and I succeeded
